reconcile/openshift_resources_base.py

```diff
@@ -308,6 +314,8 @@ def lookup_secret(
             secret_reader = SecretReader(settings)
         return secret_reader.read(secret)
     except Exception as e:
+        if allow_not_found:
+            return "SECRET_NOT_FOUND"
```

Comment:
This means this exact value needs to be tested in the template to add or not the datasource?

Reply:
@patjlm I am testing out this approach..

Reply:
I guess returning `None` would be good, right ?